.github/CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 qutebrowser/browser/history.py  @rcorre
 qutebrowser/completion/*  @rcorre
+qutebrowser/config/configdata.yml  @mschilli87
```

Comment:
I'd rather keep this somewhat sorted, but I'll fix this up after merging.

Reply:
I thought is was alphabetically sorted. That's how I placed the line.
Just now saw that `tests/` is *after* `qutebrowser/`. :smile:

Reply:
Ah, fair enough :wink: I'd rather keep it sorted by topics/people roughly, and did so in 6233358b718b73b1edbe2d80c4c8606c22855e09 @The-Compiler